packages/babel-register/test/index.js

```diff
@@ -3,7 +3,7 @@ import path from "path";
 
 const DATA_ES2015 = require.resolve("./__data__/es2015");
 
-describe("babel-register", function() {
+xdescribe("babel-register", function() {
```

Comment:
So the babel-register test also don't work with jest-mocha-runner? That was the biggest blocker for the other PR because there is no way in test to test require hooks.

So we would either need to make a special case for these tests and not run them with jest or stay with mocha.

Reply:
I'm fine with doing a special case for this if we want to use jest